docs/bdk-cli/introduction.md

```diff
@@ -0,0 +1,13 @@
+# Introduction
+
+[`bdk-cli`](https://github.com/bitcoindevkit/bdk-cli) is a light weight [`repl`](https://codewith.mu/en/tutorials/1.0/repl) wrapper over `bdk` that comes as a command line application. It is useful for quick testing and prototyping of bdk functionalities.
+
+This can also be used as a example application to create your own command line bitcoin wallet tool using bdk.
+
+`bdk-cli` can interface with all the blockchain backends supported by `bdk` currently, like `rpc`, `electrum`, `esplora` and `compact_filters`.
+
+`bdk-cli` comes with `regtest-*` features that can automatically deploy a regtest backend with a connected `bdk` wallet in background. This can be useful If you don't wanna setup your own regtest node or to try out testing quickly, especially in `repl` mode.
```

Comment:
```suggestion
`bdk-cli` comes with `regtest-*` features that can automatically deploy a regtest backend with a connected `bdk` wallet in the background. This can be useful If you don't want to set up your own regtest node or try out testing quickly, especially in `repl` mode.
```